### ISSUE TITLE
Determine license suitable for collaboration

### ISSUE THREAD

**Issue by Lukenickerson** [jayseesee/Crittermound-Fork#1]
Given the passing of [The Creator](http://www.reddit.com/user/brave_powerful_ruler), it's not clear who owns CritterMound or how it can be made open source. While the threat of a dispute is likely small, it's best to get this resolved up front.

My preference would be to see the game be released under one of the common licenses: GPL, MIT license, or even purely public domain.

**Comment by jayseesee** [comment]
I agree.  I'm not sure how we'd go about doing that, but I do feel that something along those lines is for the best.

**Comment by ClayTaeto** [comment]
I think it should be released under MIT or public domain.

**Comment by Psychemaster** [comment]
Personally I'd go with the MIT license.
